Cargo.toml

```diff
@@ -60,3 +60,6 @@ crate-type = ["rlib"]
 
 [profile.release]
 lto = "thin"
+
+[patch.crates-io]
+clvm-traits = { path = "clvm-traits" }
```

Comment:
I think a comment here is warranted. at least something brief about how this is meant to be temporary. It is, right?